ArduCopter/mode.h

```diff
@@ -184,6 +184,7 @@ class Mode {
     // pause and resume a mode
     virtual bool pause() { return false; };
     virtual bool resume() { return false; };
+    void make_safe_ground_handling(bool force_throttle_unlimited = false);
```

Comment:
Just a small thing but any reason why the declaration is moved higher?  I think it was better in its original place.

Reply:
I couldn't call it from the AC_Autotune library.  So I had to make it public.  Any ideas on how to do it otherwise?

Reply:
from within mode_autotune.cpp I think it should work if the "copter.flightmode->" prefix is removed.

Reply:
I’m pretty sure I tried that first but it doesn’t work because the code is in autotune and not mode_autotune

Reply:
@bnsgeyer,

OK, you're right that we can't easily remove the prefix without it becoming awkward.

Could you just add this comment above the function?
// handle situations where the vehicle is on the ground waiting for takeoff

Reply:
done.  Thanks!